src/functions.php

```diff
@@ -318,7 +318,7 @@ function try_fopen($filename, $mode)
     restore_error_handler();
 
     if ($ex) {
-        /** @var $ex \RuntimeException */
```

Comment:
the more standard annotation seems to reverse the arguments to `/** @var \RuntimeException $ex */`. maybe that works better for you?